### PR TITLE
[KUBOS-427] Modifications needed for telemetry/ipc

### PR DESCRIPTION
**Pull request by plauche**
* Enables networking/tcp support
* Enable SCTP
* Brings up lo on boot up

### CODE REVIEW

File: configs/at91sam9g20isis_defconfig [hunk 1]
```diff
@@ -24,6 +24,8 @@ BR2_LINUX_KERNEL_DTS_SUPPORT=y
 BR2_LINUX_KERNEL_USE_CUSTOM_DTS=y
 BR2_LINUX_KERNEL_CUSTOM_DTS_PATH="${BR2_EXTERNAL_KUBOS_LINUX_PATH}/board/kubos/at91sam9g20isis/at91sam9g20isis.dts"
 BR2_PACKAGE_BUSYBOX_CONFIG="${BR2_EXTERNAL_KUBOS_LINUX_PATH}/board/kubos/at91sam9g20isis/busybox-kubos.config"
+BR2_PACKAGE_BUSYBOX_SHOW_OTHERS=y
+BR2_PACKAGE_DEBIANUTILS=y
```
Comment (CatFreed) [inline configs/at91sam9g20isis_defconfig]: Why did you turn this on? Busybox has some support for Debian utilities already.

File: overlay/etc/network/if-pre-up.d/wait_iface [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```
Comment (CatFreed) [inline overlay/etc/network/if-pre-up.d/wait_iface]: Why was this file included? It should already be present in the default rootfs

Comment (plauche) [inline overlay/etc/network/if-pre-up.d/wait_iface]: It is included by default, however the permissions on the file were not correct, so the `run-parts` utility failed to execute the script, which meant the network interface was not brought up.

File: configs/at91sam9g20isis_defconfig (hunk 1 above)
Comment (plauche) [inline configs/at91sam9g20isis_defconfig]: We need the `run-parts` utility which is included in this package to execute certain init scripts, such as those in `/etc/network/if-pre-up.d`.

Comment (CatFreed) [inline configs/at91sam9g20isis_defconfig]: `run-parts` is also available in the busybox configuration under the 'Debian Utilities' category as `CONFIG_RUN_PARTS`. There are some other utilities already turned on there, like `which`
